docs/NOW-spec.md

```diff
@@ -1365,6 +1334,9 @@ The NOW_EXEC_RUN_MSG message is used to send a run request. This request type ma
 
 **command (variable)**: A NOW_VARSTR structure containing the command to execute.
 
+#### NOW_EXEC_CMD_MSG
+<!-- TODO: Define CMD message -->
```

Comment:
Not defined -- should be keep it? We already have `NOW_EXEC_BATCH_MSG`, what is the difference?